Cargo.toml

```diff
@@ -7,3 +7,6 @@ repository = "https://github.com/dragostis/pest"
 documentation = "http://dragostis.github.io/pest/pest/"
 keywords = ["pest", "parser", "peg", "grammar"]
 license = "MPL-2.0"
+
+[dependencies]
+clap = "2.8.0"
```

Comment:
@sunng87 I was thinking of making it a general `pest` tool, so the `pest` name would be best. I'll probably have to resort to the `pest-tool` name.